admin/Dockerfile

```diff
@@ -1,44 +1,37 @@
-# RedHat UBI 8 with nodejs 14
-FROM registry.access.redhat.com/ubi8/nodejs-14:1-75.1652296492 as builder
+# Global args
+ARG build_dir=admin
+ARG port=4200
 
-# Install packages, build and keep only prod packages
+
+# Build container
+FROM node:14.21.2-alpine3.17 AS build
+
+# Install dependencies and build static content
+ARG build_dir
 WORKDIR /app
-COPY . ./
-USER root
+COPY ./libs ./libs
+COPY ./${build_dir} ./${build_dir}
 RUN cd libs && \
     npm ci && \
-    cd ../admin && \
+    cd ../${build_dir} && \
     npm ci && \
-    npm run build:admin && \
-    rm -rf node_modules
+    npm run build:${build_dir} && \
+    rm -rf node_modules ../libs/node_modules
 
-# Deployment container
-# FROM registry.access.redhat.com/ubi8/ubi-micro
-FROM caddy:2
 
-# Vars - configured for local development
-ENV FOM_ENV=dev
+# Deploy container
+FROM caddy:2.4.6-alpine
 
-# Node packages and dependencies
-COPY --from=builder /usr/bin/node /usr/bin/
-COPY --from=builder /usr/lib64/libz.so.1 /usr/lib64/
-COPY --from=builder /usr/lib64/libbrotlidec.so.1 /usr/lib64/
-COPY --from=builder /usr/lib64/libbrotlienc.so.1 /usr/lib64/
-COPY --from=builder /usr/lib64/libcrypto.so.1.1 /usr/lib64/
-COPY --from=builder /usr/lib64/libssl.so.1.1 /usr/lib64/
-COPY --from=builder /usr/lib64/libstdc++.so.6 /usr/lib64/
-COPY --from=builder /usr/lib64/libgcc_s.so.1 /usr/lib64/
-COPY --from=builder /usr/lib64/libbrotlicommon.so.1 /usr/lib64/
-
-# Copy over app
-WORKDIR /app
-COPY --from=builder /app/admin/dist ./dist
-COPY --from=builder /app/admin ./admin
-COPY --from=builder /app/admin/Caddyfile .
+# Envar
+ENV FOM_ENV=dev
```

Comment:
Why are we using envar FOM_ENV=dev for DEV, TEST and PROD?  It's even in the OpenShift templates.  Is it something we missed earlier?
@basilv @ianliuwk1019

Reply:
I checked the code base and am confident that FOM_ENV isn't used (must be left over).